package.json

```diff
@@ -4,11 +4,17 @@
   "main": "index.js",
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "^7.0.0-beta.56",
-    "@babel/preset-env": "^7.0.0-beta.56",
-    "@babel/preset-react": "^7.0.0-beta.56",
-    "babel-loader": "^8.0.0-beta",
+    "@babel/core": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-0",
```

Comment:
Were the babel packages upgraded by dependency of jest/enzyme?

Reply:
No, I manually updated the package versions for babel. When I tried installing `babel-jest` with the versions of the babel packages that we had before, it gave a warning about an unmet peer dependency for babel. I was following this step: https://jestjs.io/docs/en/getting-started#using-babel